data/dataexchange/src/test/java/tech/pegasys/teku/data/SlashingProtectionExporterTest.java

```diff
@@ -145,6 +147,7 @@ public void shouldRequirePubkeyInFilename(@TempDir Path tempDir)
   }
 
   @Test
+  @DisabledOnOs(WINDOWS)
```

Comment:
Is there some equivalent test that we could run when building on Windows? Same question for the other disabled test.

Reply:
Yep, found the solution for Win 👍